mne/viz/topomap.py

```diff
@@ -1184,7 +1184,14 @@ def plot_tfr_topomap(tfr, tmin=None, tmax=None, fmin=None, fmax=None,
     if not show_names:
         names = None
 
-    data = tfr.data
+    data = tfr.data[picks, :, :]
+
+    # merging grads before rescaling makes ERDs visible
+    if merge_grads:
+        from ..channels.layout import _merge_grad_data
+        data, shape = _merge_grad_data(data), data.shape
```

Comment:
so `_merge_grad_data` does not retain the shape properly? does it make this 3D array 2D? If so perhaps it should be fixed to make it properly return an array of the same shape (other than along the merged dimension)